src/openssl.c

```diff
@@ -1785,6 +1789,359 @@ static size_t compat_SSL_SESSION_get_master_key(const SSL_SESSION *session, unsi
 }
 #endif
 
+#if !HAVE_SSL_CIPHER_STANDARD_NAME
+static const struct {int num; const char *name;} ssl_ciphers_tbl[] = {
+	{0x0000, "SSL_NULL_WITH_NULL_NULL"},
+	{0x0001, "SSL_RSA_WITH_NULL_MD5"},
+	{0x0002, "SSL_RSA_WITH_NULL_SHA"},
+	{0x0003, "SSL_RSA_EXPORT_WITH_RC4_40_MD5"},
+	{0x0004, "SSL_RSA_WITH_RC4_128_MD5"},
+	{0x0005, "SSL_RSA_WITH_RC4_128_SHA"},
+	{0x0006, "SSL_RSA_EXPORT_WITH_RC2_CBC_40_MD5"},
+	{0x0007, "SSL_RSA_WITH_IDEA_CBC_SHA"},
+	{0x0008, "SSL_RSA_EXPORT_WITH_DES40_CBC_SHA"},
+	{0x0009, "SSL_RSA_WITH_DES_CBC_SHA"},
+	{0x000A, "SSL_RSA_WITH_3DES_EDE_CBC_SHA"},
+	{0x000B, "SSL_DH_DSS_EXPORT_WITH_DES40_CBC_SHA"},
+	{0x000C, "SSL_DH_DSS_WITH_DES_CBC_SHA"},
+	{0x000D, "SSL_DH_DSS_WITH_3DES_EDE_CBC_SHA"},
+	{0x000E, "SSL_DH_RSA_EXPORT_WITH_DES40_CBC_SHA"},
+	{0x000F, "SSL_DH_RSA_WITH_DES_CBC_SHA"},
+	{0x0010, "SSL_DH_RSA_WITH_3DES_EDE_CBC_SHA"},
+	{0x0011, "SSL_DHE_DSS_EXPORT_WITH_DES40_CBC_SHA"},
+	{0x0012, "SSL_DHE_DSS_WITH_DES_CBC_SHA"},
+	{0x0013, "SSL_DHE_DSS_WITH_3DES_EDE_CBC_SHA"},
+	{0x0014, "SSL_DHE_RSA_EXPORT_WITH_DES40_CBC_SHA"},
+	{0x0015, "SSL_DHE_RSA_WITH_DES_CBC_SHA"},
+	{0x0016, "SSL_DHE_RSA_WITH_3DES_EDE_CBC_SHA"},
+	{0x0017, "SSL_DH_anon_EXPORT_WITH_RC4_40_MD5"},
+	{0x0018, "SSL_DH_anon_WITH_RC4_128_MD5"},
+	{0x0019, "SSL_DH_anon_EXPORT_WITH_DES40_CBC_SHA"},
+	{0x001A, "SSL_DH_anon_WITH_DES_CBC_SHA"},
+	{0x001B, "SSL_DH_anon_WITH_3DES_EDE_CBC_SHA"},
+	{0x001D, "SSL_FORTEZZA_KEA_WITH_FORTEZZA_CBC_SHA"},
+	{0x001E, "SSL_FORTEZZA_KEA_WITH_RC4_128_SHA"},
```

Comment:
Should this be `TLS_KRB5_WITH_DES_CBC_SHA`? See https://github.com/openssl/openssl/pull/1743/files and https://github.com/daurnimator/lua-http/pull/116#issuecomment-404753518